### DISCUSSION TITLE
Likelihood profile on initial F

### DISCUSSION THREAD

**Discussion by N-DucharmeBarth-NOAA** [nmfs-stock-synthesis/stock-synthesis#538] [Q&A] [answered]
Hello, I have a Stock Synthesis model which estimates initial F. I'd like to run a likelihood profile on this parameter to see what component of the likelihood contributes the most to this parameter estimate. However, I'm not sure that I've set things up correctly since there seem to be some strange things going on with the survey catchabilities and equilibrium catch component likelihood. Are there any additional aspects of the model or control file that need to be taken into account when profiling over this parameter?
Best,
Nicholas

**Comment by Rick-Methot-NOAA** [comment, marked as answer]
Hi Nicholas.
I assume you mean the initial equilibrium F parameter and that your model includes data for initial equilibrium catch by including catch for year -999.  In this case, it is that datum that dominates the estimation of this parameter, conditional on the value for ln(R0).  This parameter also has a large impact on the slope of the initial population age composition so a big impact on fitting composition data early in the time series.
This calculation is an equilibrium calculation and uses exactly the same code as the MSY calculation.  I suspect you are profiling to too high of a F level and reducing the population to a very low level before the time series starts.
In fact, if you turn this spawner-recruitment switch to "1"
0  # 0/1 to use steepness in initial equ recruitment calculation
Then profiling to F > Fmsy will be logically impossible.
Rick

**Comment by iantaylor-NOAA** [comment]
Adding to Rick's comment, It's possible that the equilibrium catch is dominates the estimation of equilibrium F so much that the model is having convergence issues as you move away from the MLE for that parameter. If that's the case, you could increase the uncertainty in the equilibrium catch, or apply a small lambda value to component 9: https://nmfs-ost.github.io/ss3-doc/SS330_User_Manual_release.html#lambdas-emphasis-factors.

However, note that reducing the influence of that datum would change the value of equilibrium F that minimizes the total negative log-likelihood (but the profiles over all the other likelihood components should remain unchanged).

**Comment by N-DucharmeBarth-NOAA** [comment]
Thank you very much Rick and Ian for your comments. I'll give it another try based on your suggestions.
